### PR TITLE
DELIA-60164: Add support for custom process name

### PR DESCRIPTION
**Pull request by filipe-norte-red**
Reason for change: Multiple web processes with same name make correct process identification harder
Test Procedure: Verify process name changes when
custom name provided
Risks: Low

### CODE REVIEW

File: WebKitBrowser/JSPP.config [hunk 1]
```diff
@@ -124,6 +124,10 @@ map()
             kv(name "XDG_DATA_HOME")
             kv(value "/opt/QT/home")
         end()
+        map()
+            kv(name "PROCESS_NAME")
+            kv(value "JSPP-WPEWebProcess")
```
Comment (emutavchi) [inline WebKitBrowser/JSPP.config:129]: Let's name it JSPPWebProcess, just to keep name short.

**Review by emutavchi — COMMENTED** [review]
Otherwise looks good to me

File: WebKitBrowser/JSPP.config (hunk 1 above)
Comment (filipe-norte-red) [reply WebKitBrowser/JSPP.config]: Changed.

**Review by emutavchi — APPROVED** [review]

**Review by modeveci — APPROVED** [review]
It is ok for me.

**Pull request merged**